DSharpPlus.Commands/Processors/TextCommands/TextCommandProcessor.cs

```diff
@@ -16,16 +17,26 @@
 
 namespace DSharpPlus.Commands.Processors.TextCommands;
 
+/// <summary>
+/// Handles the processing of text-based commands executed via Discord messages.
+/// </summary>
 public sealed class TextCommandProcessor(TextCommandConfiguration? configuration = null) : BaseCommandProcessor<MessageCreatedEventArgs, ITextArgumentConverter, TextConverterContext, TextCommandContext>
 {
+    /// <summary>
+    /// The intents required for this processor to function.
+    /// </summary>
     public const DiscordIntents RequiredIntents = DiscordIntents.DirectMessages // Required for commands executed in DMs
                                                 | DiscordIntents.GuildMessages; // Required for commands that are executed via bot ping

```

Comment:
nitpick: this is technically inaccurate, only one of the two is ever required

Reply:
Which one isn't required

Reply:
neither is required individually, it's just that one of them must be present, no matter which one